tests/ticket2986.py

```diff
@@ -46,8 +46,10 @@ def test(self):
 
         new = ccds.addNew()
         new.set("id", 0)
-        new.set("bbox.min", afwGeom.Point2I(0,0))
-        new.set("bbox.max", afwGeom.Point2I(1024,1024))
+        new.set("bbox_min_x", 0)
+        new.set("bbox_min_y", 0)
+        new.set("bbox_max_x", 1024)
+        new.set("bbox_max_y", 1024)
 
```

Comment:
If this wasn't test code, I'd agree, and create a pair of Point FunctorKeys - and in the long run, we probably want a Box FunctorKey, though that's beyond the scope of this issue.

But even if we had that, I don't it'd be more concise than what's already there, and in this context the performance advantage of setting them at the same time goes away.